k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/K8sConfig.kt

```diff
@@ -15,7 +15,9 @@ const val K8S_PROVIDER = "kubernetes"
 const val SOURCE_TYPE = "text"
 
 const val K8S_LAST_APPLIED_CONFIG: String = "kubectl.kubernetes.io/last-applied-configuration"
-const val KEEL_MONIKER_APP: String = "moniker.spinnaker.io/application"
+const val KEEL_MONIKER_APP_ANNOTATION: String = "moniker.spinnaker.io/application"
```

Comment:
This doesn't seem to be used. What is this for?

Reply:
yeah I dont think we use it. I can remove it.